omero/developers/Python.rst

```diff
@@ -459,10 +543,12 @@ Write data
 
 ::
 
-    project = conn.getObject("Project", projectId)
     link = omero.model.ProjectDatasetLinkI()
-    link.setParent(omero.model.ProjectI(project.getId(), False))
-    link.setChild(dataset_obj)
+    # We can use a 'loaded' object, but we might get an Exception
+    # link.setChild(dataset_obj)
+    # Better to use an 'unloaded' objects (loaded = False)
```

Comment:
```suggestion
    # Better to use an 'unloaded' object (loaded = False)
```

Reply:
(or drop "an")

Reply:
I can't merged this. Leaving to you, @will-moore.